docs/cloud-native-security/cspm-get-started-gcp.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[cspm-get-started-gcp]]
+= Get started with CSPM for GCP
+
+[discrete]
+[[cspm-overview-gcp]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) feature.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices:
+** `logs-cloud_security_posture.findings_latest-*`
+** `logs-cloud_security_posture.scores-*`
+** `Logs-cloud_security_posture.findings`
+* The user who gives the CSPM integration GCP permissions must be a GCP project `admin`.
+--
+
+[discrete]
+[[cspm-setup-gcp]]
+== Set up CSPM for GCP
+
+To set up CSPM for GCP, first add the CSPM integration, then enable cloud account access.
+
+
+[discrete]
+[[cspm-add-and-name-integration-gcp]]
+=== Add your CSPM integration
+. From the Elastic Security *Get started* page, click *Add integrations*.
+. Search for `CSPM`, then click on the result.
+. Click *Add Cloud Security Posture Management (CSPM)*.
+. Give your integration a name that matches the purpose or team of the GCP account you want to monitor, for example, `dev-gcp-project`.
+
+[discrete]
+[[cspm-set-up-cloud-access-section-gcp]]
+=== Set up cloud account access
+The CSPM integration requires admin access to your GCP project to
+
+For most users, the simplest option is to use a Google Cloud Shell script to automatically provision the necessary resources and permissions in your GCP account. This method, as well as two manual options, are described below.
+
+[discrete]
+[[cspm-set-up-cloudshell]]
+=== Cloud Shell script setup (recommended)
+
+. Under **Setup Access** select **Google Cloud Shell**.
+. Under **Where to add this integration**,
```

Comment:
@benironside, a new compute instance will always get created for elastic agent to run on if they use this automated option, so the user should always select new hosts- the other option won't work.